griptape/config/azure_openai_structure_config.py

```diff
@@ -69,7 +69,7 @@ class AzureOpenAiStructureConfig(StructureConfig):
     )
     image_query_driver: BaseImageQueryDriver = field(
         default=Factory(
-            lambda self: AzureOpenAiVisionImageQueryDriver(
+            lambda self: AzureOpenAiImageQueryDriver(
```

Comment:
can you update the model to `gpt-4o` here too since its available?

Reply:
Sick, will do.

Reply:
as well as the prompt driver too

Reply:
do we need a similar Azure deployment note [here](https://github.com/griptape-ai/griptape/blob/2aed4eb37c5349c720a5d6c3cb7d77590db4c2c8/docs/griptape-framework/drivers/image-query-drivers.md?plain=1#L92) for GPT-4o?